.github/workflows/update-image-tag-api-dev.yaml

```diff
@@ -67,3 +71,50 @@ jobs:
         git commit -m "Update API image tag to $NEW_TAG"
 
         git push
+
+    # Checkout the repository
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    # Clone the wiki repository
+    - name: Clone wiki repository
+      run: |
+        echo "Cloning wiki repo https://github.com/$GITHUB_REPOSITORY.wiki.git"
+        git clone "https://$GITHUB_ACTOR:$GH_TOKEN@github.com/$GITHUB_REPOSITORY.wiki.git" ./wiki
+
+    # Setup Python
+    - name: Setup Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.10' 
+
+    # Run the update wiki Python script
+    - name: Run update wiki Python script
+      run: python ./.github/helpers/update-wiki-table.py ./wiki/Image-tags.md "API V2" "Latest Build Image Tag" "${{ steps.get-pr-number.outputs.result }}"
+
+    # Commit and push changes to the wiki
+    - name: Commit and push changes to wiki
+      run: |
+        cd ./wiki
+        git config user.email "adam.kroon@gov.bc.ca"
```

Comment:
Not using github actor here?

Reply:
was thinking of consistency with the previous job. will change.